index.js

```diff
@@ -24,12 +24,30 @@ function getLocale(string) {
 	return (string && string.replace(/[.:].*/, ''));
 }
 
+function getLocales() {
+	return execa.stdout('locale', ['-a']);
+}
+
+function getLocalesSync() {
+	return execa.sync('locale', ['-a']).stdout;
+}
+
+function getSupportedLocale(locale, locales = '') {
+	return locales.indexOf(locale) === -1 ? defaultLocale : locale;
```

Comment:
`indexOf` => `includes`

Reply:
I made this change. According to the [package.json](https://github.com/sindresorhus/os-locale/blob/master/package.json#L13) we require engine `>=6` however from [node.green](https://node.green/#ES2015-built-ins-well-known-symbols-Symbol-match--String-prototype-includes), this function is not available in `6.4.0`. However, users should always prefer the latest semver minior/patch of each release, i just mentioned this :)